SofaKernel/framework/sofa/core/objectmodel/BaseObject.cpp

```diff
@@ -244,36 +238,22 @@ void BaseObject::copyAspect(int destAspect, int srcAspect)
 {
     Base::copyAspect(destAspect, srcAspect);
     // copyAspect is no longer recursive to slave objects
-    /*
-        for(VecSlaves::const_iterator iSlaves = l_slaves.begin(); iSlaves != l_slaves.end(); ++iSlaves)
-        {
-            (*iSlaves)->copyAspect(destAspect, srcAspect);
-        }
-    */
 }
 
 /// Release memory allocated for the specified aspect.
 void BaseObject::releaseAspect(int aspect)
 {
     Base::releaseAspect(aspect);
     // releaseAspect is no longer recursive to slave objects
-    /*
-        for(VecSlaves::const_iterator iSlaves = l_slaves.begin(); iSlaves != l_slaves.end(); ++iSlaves)
-        {
-            (*iSlaves)->releaseAspect(aspect);
-        }
-    */
 }
 
 void BaseObject::init()
 {
-
-
 	for(VecData::const_iterator iData = this->m_vecData.begin(); iData != this->m_vecData.end(); ++iData)
 	{
 		if ((*iData)->isRequired() && !(*iData)->isSet())
 		{
-            serr << "Required data \"" << (*iData)->getName() << "\" has not been set. (Current value is " << (*iData)->getValueString() << ")" << sendl;
+            msg_error() << "Required data \"" << (*iData)->getName() << "\" has not been set. (Current value is " << (*iData)->getValueString() << ")" ;
```

Comment:
Here is the problematic line that cause scene errors. 
in previous code 'serr' is actually emitting a warning (I already told all the bad I think about that ;))
I replaced that with msg_error which is wrong.